src/management-system-v2/components/bpmn-timeline.tsx

```diff
@@ -0,0 +1,92 @@
+'use client';
+
+import { updateProcess } from '@/lib/data/processes';
+import Modeler from 'bpmn-js/lib/Modeler';
+import type ElementRegistry from 'diagram-js/lib/core/ElementRegistry';
+import type ElementFactory from 'diagram-js/lib/core/ElementFactory';
+import type Canvas from 'diagram-js/lib/core/Canvas';
+import Modeling from 'diagram-js/lib/features/modeling/Modeling';
+import { useEnvironment } from './auth-can';
+import { Element, Parent } from 'diagram-js/lib/model/Types';
+import { useEffect, useRef, useState } from 'react';
+import useTimelineViewStore from '@/app/(dashboard)/[environmentId]/processes/[processId]/use-timeline-view-store';
```

Comment:
You can move the store into the lib folder, if it is being used in multiple locations. This static import path to the page folder is more at risk of change in the future.